app/assets/stylesheets/admin.scss

```diff
@@ -29,7 +29,7 @@ body > div.container {
 
 .users {
   li.user {
-    border-bottom: 1px solid $light-grey;
+    border-bottom: 1px solid $text-grey;
```

Comment:
`new_styles/new_mixins` overwrote `$light-grey` on the desktop version with the same value `$text-grey` has. Because that was the only important thing `new_styles/new_mixins` did I removed the file and changed this variable.